src/history/historyTracker.ts

```diff
@@ -441,6 +442,24 @@ export class HistoryTracker {
       editor: isUppercaseMark ? vscode.window.activeTextEditor : undefined,
     };
     this.putMarkInList(newMark);
+    this.updateMarksDecorators();
```

Comment:
Let's separate the data updating from the view updating, so that if the marks are updated elsewhere (`:delmarks`, for instance), we don't have to remember to update the view.
https://github.com/VSCodeVim/Vim/blob/3d9718d70debe881751ddf37832995b1100c1ee0/src/mode/modeHandler.ts#L1347

Reply:
done